src/index.js

```diff
@@ -32,9 +51,8 @@ const createStyled = (jss?: Function = jssDefault) => (baseStyles: Object = {})
     const staticTag = `${tag}-${++counter}`
 
     return class StyledElement extends PureComponent {
-      static tag = tag
-
-      static styles = elementStyles
+      static tag: string = tag
+      static styles: ComponentStylesType = elementStyles
```

Comment:
just realized: elementStyles should be elementStyle, ownStyles should be ownStyle